src/components/Navbar/index.js

```diff
@@ -0,0 +1,68 @@
+import {Link} from 'react-router-dom';
+
+import StyledNavBar from './styled';
+
+export default function NavBar() {
+	return (
+		<StyledNavBar>
+			<Link to="/questions">
```

Comment:
Thanks for the hint :) I will make a whole accessibility user Story at the end 🕺🏻

Reply:
That would be one way to approach it, however, it might be better to give the topic some space and time in every story as you then don't have to change a lot in the end 😉. But that's just my personal opinion.